drivers/led/ht16k33.c

```diff
@@ -0,0 +1,282 @@
+/*
+ * Copyright (c) 2019 Henrik Brix Andersen <henrik@brixandersen.dk>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief LED driver for the HT16K33 I2C LED driver with keyscan
+ */
+
+#include <i2c.h>
+#include <led.h>
+#include <zephyr.h>
+
+#define LOG_LEVEL CONFIG_LED_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(ht16k33);
+
+#include "led_context.h"
+
+/* HT16K33 commands and options */
+#define HT16K33_CMD_DISP_DATA_ADDR 0x00
+#define HT16K33_CMD_SYSTEM_SETUP   0x20
+#define   HT16K33_OPT_S            BIT(0)
```

Comment:
Please keep the alignment for all defines

Reply:
Fixed.